scripts/cibuild.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+set -e
+
+vagrant up --provision
+vagrant ssh -c "
+    cd /vagrant
+    echo -e '\nRunning linter...'
+
+    # run linter with console output
+    npm run lint
+
+    # run linter with file output for Jenkins
+    npm run lint:ci
```

Comment:
For your reference, [these](https://wiki.jenkins.io/display/JENKINS/Building+a+software+project#Buildingasoftwareproject-belowJenkinsSetEnvironmentVariables) are the ENV variables set by Jenkins, but I think it's safe to assume that this script is only running in CI. One other note though: in [other projects](https://github.com/azavea/raster-foundry/blob/develop/scripts/cibuild#L24) we've included a conditional to only run a script if it's been invoked directly,  (i.e. not via `source`).